awspricing/cache.py

```
@@ -36,7 +36,7 @@ def cache_path():
         if not os.path.isdir(setting):
             try:
                 os.makedirs(setting)
-            except:
+            except OSError:
```

Good catch. Bad me doing a bare except :)